app/views/forms/incident-report/_intro.html.erb

```diff
@@ -0,0 +1,2 @@
+<li><strong>EMERGENCY</strong> -&nbsp; call <strong>1-1234</strong>&nbsp;</span>&nbsp; &nbsp;[Temple Campus Police 215-204-1234]</li>
```

Comment:
List items should always be within a list element.

Reply:
Fixed